content/en/docs/tasks/traffic-management/egress/wildcard-egress-hosts/index.md

```diff
@@ -298,8 +289,12 @@ The SNI proxy will forward the traffic to port `443`.
 
 1.  The following command will generate `istio-egressgateway-with-sni-proxy.yaml` which you can optionally edit and then deploy.
 
+    {{< warning >}}
+    Note that the configuration below is currently supported for Helm only, not supported for Istioctl.
+    {{< /warning >}}
+
     {{< text bash >}}
-    $ cat <<EOF | istioctl manifest generate --set values.global.istioNamespace=istio-system -f - > ./istio-egressgateway-with-sni-proxy.yaml
+    $ cat <<EOF | helm template install/kubernetes/helm/istio/ --name istio-egressgateway-with-sni-proxy --namespace istio-system -x charts/gateways/templates/deployment.yaml -x charts/gateways/templates/service.yaml -x charts/gateways/templates/serviceaccount.yaml -x charts/gateways/templates/autoscale.yaml -x charts/gateways/templates/role.yaml -x charts/gateways/templates/rolebindings.yaml --set global.istioNamespace=istio-system -f - > ./istio-egressgateway-with-sni-proxy.yaml
```

Comment:
Our docs are not supposed to use helm anymore, since it is in the processes of being deprecated.

Reply:
Handled in https://github.com/istio/istio.io/pull/6217/commits/0a32cd9e5fd0ce0598484beb7ff3c7d6769d5977 using IstioOperator.